docs/Common/Structs/index.md

```diff
@@ -487,7 +487,6 @@
 |radioEnable|Boolean|false||True if the radio is on, false is the radio is off|
 |state|Common.RadioState|false||||
 |sisData|Common.SisData|false||Read-only Station Information Service (SIS) data provides basic information about the station such as call sign, as well as information not displayable to the consumer such as the station identification number|
-|hdRadioEnable|Boolean|false||True if the hd radio is on, false is the radio is off|
```

Comment:
Why was this deleted?

Reply:
@JackLivio, solving conflict I removed the parameter thinking it is not related to status light values.
after investigation, it turned out the param was added and merged by earlier #pull/106.
added back https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/107/commits/f917f54c779c148ec6928936f871d7b60f4d6150